stylelint-polaris/plugins/coverage/index.js

```diff
@@ -55,10 +55,79 @@ module.exports = stylelint.createPlugin(
           },
         );
       }
+
+      const disabledCoverageWarnings =
+        result.stylelint.disabledWarnings?.filter((disabledWarning) =>
+          disabledWarning.rule.startsWith(ruleName),
+        );
+
+      if (!disabledCoverageWarnings?.length) return;
+
+      const disabledCoverageLines = Array.from(
+        new Set(disabledCoverageWarnings.map(({line}) => line)),
+      );
+
+      // Ensure all stylelint-polaris/coverage disable comments
+      // have a description prefixed with "polaris:"
+      for (const disabledRange of result.stylelint.disabledRanges.all) {
+        if (
+          !isDisabledCoverageRule(disabledCoverageLines, disabledRange) ||
+          disabledRange.description?.startsWith('polaris:')
+        ) {
+          continue;
+        }
+
+        stylelint.utils.report({
+          message: `Missing "polaris:" prefix in disable comment description`,
```

Comment:
Maybe we could generalize this message to educate the user when they don't add any comment at all like in [this](https://github.com/Shopify/polaris/pull/7589/files#diff-6c0661b0cc3928ccb89867e1d14893f8e7b98d824996bcd117cb6ccae9954bd0R61) test case and have an example of what they should be doing, i.e.

```suggestion
          message: `Incorrect disable comment for stylelint-polaris. The correct format is: "/* ${disableComment} -- polaris: {Context on why the rule is disabled}  */"`,
```

Hopefully you could get the `disableComment` from the `disabledRange` object to print out either `stylelint-disable-next-line` or `stylelint-disable`. 

Hopefully this makes sense!

Reply:
Thanks for the feedback! We wen't with option 1: https://github.com/Shopify/polaris/pull/7589/commits/3b97ef0fa17f651f65f85be8d267c05dead54aed